src/lib/HST/Util/Messages.hs

```diff
@@ -35,6 +36,9 @@ data Message = Message { msgSeverity :: Severity
 message :: Severity -> S.SrcSpan a -> String -> Message
 message severity srcSpan = Message severity (S.toMsgSrcSpan srcSpan)
 
+messageWithoutSrcSpan :: Severity -> String -> Message
+messageWithoutSrcSpan severity = Message severity Nothing
```

Comment:
I would have chosen a shorter name (`messageNoSrc` for example), but this is fine as well. You should add a Haddock comment for this function, though.